stable/artifactory-ha/README.md

```diff
@@ -1019,6 +1049,174 @@ helm upgrade --install artifactory-ha -f filebeat.yaml --namespace artifactory-h
 
 This will start sending your Artifactory logs to the log aggregator of your choice, based on your configuration in the `filebeatYml`
 
+### Install Artifactory HA with Nginx and Terminate SSL in Nginx Service(LoadBalancer).
+To install the helm chart with performing SSL offload in the LoadBalancer layer of Nginx.
+For Ex: Using AWS ACM certificates to do SSL offload in the loadbalancer layer.
+
+```bash
+helm upgrade --install artifactory-ha \
```

Comment:
Can we change this to a yaml block and not command line?

Reply:
Done

Reply:
Done